docs/topics/db/models.txt

```diff
@@ -1238,6 +1238,11 @@ number of proxy models that share a common non-abstract parent class.
     In earlier versions, a proxy model couldn't inherit more than one proxy
     model that shared the same parent class.
 
+.. note::
+
+    The ``ordering`` and ``get_latest_by`` meta attributes are still inherited
```

Comment:
I've looked into the code and this was introduced at 9c52d56f6f8a9cdafb231adf9f4110473099c9b5. This `Meta` inheritance is tied to _non abstract_ models and, thus, not only for proxy models. These meta attributes are defined [here](https://github.com/django/django/blob/master/django/db/models/base.py#L139) and proxy setup is [done afterwards](https://github.com/django/django/blob/master/django/db/models/base.py#L167).